media-automount

```diff
@@ -32,6 +32,12 @@ else
     dev=/dev/${1##*/}
 fi
 
+if [ "$dev" == "$(findmnt -n -o SOURCE /)" ]
```

Comment:
There should only be one '=' not two. It works with this change.
([test man](https://linux.die.net/man/1/test))

Reply:
Good catch, thanks.